### ISSUE TITLE
List of figures breaks if longer than two pages.

### ISSUE THREAD

**Issue by rlabuonora** [rstudio/pagedown#291]
Hi! Thanks a lot for this terrific package!

I'm trying to have a big list of figures but Chrome fails to render it.
 

``````rmarkdown

---
title: "A Multi-page HTML Document"
author: "Yihui Xie and Romain Lesur"
date: "`r Sys.Date()`"
output:
  pagedown::html_paged:
    toc: true
    self_contained: false
lof: true # insert a list of figures
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
``` 

```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```

```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```

```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```
```{r, fig.cap = 'A very simple plot'}
plot(1)
```

 

```{r, fig.cap = 'A very simple plot'}
plot(1)
```

``````

The browser console shows a warning:

`
Unable to layout item:  <li data-ref=​"0ba4178e-0831-4cb6-8f97-b0830fba66bd">​…​</li>​
renderTo @ paged.js:1354
`
Enabling knit: pagedown::chrome_print results in the following error:

```
Warning: A runtime exception has occured while executing JavaScript
  Runtime exception message:
    TypeError: Cannot read properties of null (reading 'parentElement')
    at findPage (http://127.0.0.1:3190/reporte_postales_files/paged-0.18.1/js/config.js:62:15)
    at tocEntriesInfos (http://127.0.0.1:3190/reporte_postales_files/paged-0.18.1/js/config.js:92:20)
    at window.PagedConfig.after (http://127.0.0.1:3190/reporte_postales_files/paged-0.18.1/js/config.js:122:24)
```

Any ideas on how to solve this are greatly appreciated!

**Comment by jtrecenti** [comment]
Same issue with TOC and Chrome:

Works:

```
---
title: "A Multi-page HTML Document"
author: "Yihui Xie and Romain Lesur"
date: "`r Sys.Date()`"
output:
  pagedown::html_paged:
    toc: true
---

# Super
# Big
# Table
# Of
# Contents
# For
# My
# Awesome
# pagedown
# document
# rendering
# awesome
# reports
# with
# R
# and
# markdown
# running
# R stats
# version
# number
# 4.2
# and
# pagedown
# version
# 0.18.1
# but
# I
# don't
# think
# this
```

Does not work:

```
---
title: "A Multi-page HTML Document"
author: "Yihui Xie and Romain Lesur"
date: "`r Sys.Date()`"
output:
  pagedown::html_paged:
    toc: true
---

# Super
# Big
# Table
# Of
# Contents
# For
# My
# Awesome
# pagedown
# document
# rendering
# awesome
# reports
# with
# R
# and
# markdown
# running
# R stats
# version
# number
# 4.2
# and
# pagedown
# version
# 0.18.1
# but
# I
# don't
# think
# this
# is
```

**Comment by beatrizmilz** [comment]
Hi! I'm using pagedown and facing the same issue. Does anyone know how to fix it?

**Comment by rlabuonora** [comment]
I worked around this problem with something based on this code:

https://www.garrickadenbuie.com/blog/add-a-generated-table-of-contents-anywhere-in-rmarkdown/

**Comment by a-maldet** [comment]
I think this issue (for TOC, LOT and LOF) is resolved since `pagedown v0.19`.

**Comment by cderv** [comment]
Thanks @a-maldet ! 

I can confirm that I cannot reproduce the issue anymore.
